### PR TITLE
Prac 05 code review request

### PR DESCRIPTION
**Pull request by RhysBush**
Hello, sorry that this is late!
Please have a look through my code - I look forward to all feedback. @BrockAllen-au

### CODE REVIEW

File: prac_05/word_occurrences.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+word_to_occurrence = {}
+text = input("Text: ")
+words = text.split()
+for word in words:
+    frequency = word_to_occurrence.get(word, 0)
+    word_to_occurrence[word] = frequency + 1
```
Comment (BrockAllen-au) [inline prac_05/word_occurrences.py:6]: This works, however, you could make this a one liner like Lindsays prac video:
word_to_occurrence[word] = word_to_occurrence.get(word, 0) + 1

File: prac_05/emails.py [hunk 2]
```diff
@@ -14,7 +14,9 @@ def main():
 
 
 def get_name_from_email(email):
-    name = email
+    prefix = email.split('@')[0]
+    parts = prefix.split('.')
```
Comment (BrockAllen-au) [inline prac_05/emails.py:18]: Can also make this a one liner if you wanted to:
```parts = email.split("@")[0].split(".")```

**Review by BrockAllen-au — COMMENTED** [review]
Nice programming. Some very minor things mostly to think about :-)

File: prac_05/hex_colours.py [hunk 3]
```diff
@@ -1,6 +1,8 @@
...
+COLOUR_CODES = {"Brick Red": "#cb4154", "Burnt Orange": "#cc5500", "Lemon Yellow": "#fff44f", "Mint Green": "98ff98",
+                "Alice Blue": "#f0f8ff", "Indigo": "#4b0082", "Violet": "#d02090", "Dutch White": "#efdfbb",
+                "Black Coffee": "#3b2f2f", "Ash Grey": "#b2beb5"}
 
-colour_name = input("Enter a colour name: ")
+colour_name = input("Enter a colour name: ").title()
 while colour_name != "":
     print(f"{colour_name} has a hexadecimal value of {COLOUR_CODES.get(colour_name)}")
-    colour_name = input("Enter a colour name: ")
+    colour_name = input("Enter a colour name: ").title()
```
[the remark this answers is not in the document]
Comment (RhysBush) [reply prac_05/hex_colours.py:8]: Interesting! Definitely something to consider - formatting.

File: prac_05/word_occurrences.py (hunk 1 above)
Comment (RhysBush) [reply prac_05/word_occurrences.py:6]: list comprehension - good find!

File: prac_05/emails.py (hunk 2 above)
Comment (RhysBush) [reply prac_05/emails.py:18]: Thanks. I agree with this.

**Pull request merged**